soc/xtensa/nxp_adsp/imx8ulp/include/soc/platform.h

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
This is not needed anymore - it was specific to SOF, but not used anymore - please remove it

Reply:
Removed the file